packages/fx-core/src/core/middleware/utils/fileReadWriter.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
maybe we can call this file `v3MigrationUtils.ts` or other name, because it not only contains read write operation.

Reply:
for sure